organizations/invest-in-open-infrastructure.md

```diff
@@ -0,0 +1,34 @@
+# Invest in Open Infrastructure (IOI) Overview
+
+- **Website:** [Invest in Open Infrastructure](https://investinopen.org/)
+- **Initiatives:** [Research & Strategic Support](https://investinopen.org/our-impact/)
+
+## Overview
+
+Invest in Open Infrastructure (IOI) is a non-profit initiative committed to improving the sustainability and availability of open infrastructure. IOI works globally to provide strategic funding insights, research, and advocacy to ensure that the tools and platforms supporting open science and scholarship remain accessible, reliable, and community-governed.
+
+IOI plays a pivotal role in strengthening the open ecosystem through cross-sector partnerships, needs assessments, and capacity-building projects. While IOI does not build open source tools directly, its research and investments support open source projects and organizations across research and education.
+
+## Key Activities Supporting Open Infrastructure
+
+- **Strategic Support for Open Source Projects:** IOI provides analysis, funding pathways, and infrastructure guidance to developers and institutions involved in open source technology.  
+  [See IOI's Work](https://investinopen.org/our-impact/)
+
+- **Investments in Open Infrastructure Research:** IOI conducts global research to identify gaps in open science infrastructure and promote equitable access to funding and governance.  
+  [Read the Landscape Scan](https://investinopen.org/research/)
+
+- **Cross-Institutional Collaboration:** IOI engages with universities, libraries, and funders to advocate for more sustainable support for open infrastructure and software.  
+  [See Collaboration in Action](https://investinopen.org/blog/invest-in-open-infrastructure-announces-2-7m-commitment-from-founding-group-for-the-ioi-fund-for-network-adoption)
+
+## Impact
```

Comment:
I don't think this section adds anything that wasn't said above. I would remove it.